docs/using-mycroft-ai/get-mycroft/linux.md

```diff
@@ -29,7 +29,7 @@ This section of documentation assumes the following:
 
 Whilst Mycroft runs on a Raspberry Pi 3B or above, this is achieved through a custom release of Raspbian Lite significantly reducing the system overhead by not running a desktop environment and other unnecessary processes. Mycroft will run on older hardware however your experience may vary significantly.
 
-Our [Precise wake word engine](https://github.com/MycroftAI/mycroft-precise#mycroft-precise) also relies upon TensorFlow. For x86 Intel processors this requires the AVX \(Advanced Vector Extensions\) instruction set. To ensure your system supports AVX open a terminal and run: `grep avx /proc/cpuinfo`. AVX should be listed under the flags for each CPU core. If nothing is returned it is most likely that your system does not support AVX. Technical users may be able to build an older version of TensorFlow \(1.13\) from source using the [instructions provided on their website](https://www.tensorflow.org/install/source). Alternatively you may use Mycroft with the PocketSphinx wake word engine.
+Our [Precise wake word engine](https://github.com/MycroftAI/mycroft-precise#mycroft-precise) also relies upon TensorFlow. For x86 Intel processors this requires the AVX \(Advanced Vector Extensions\) instruction set. To ensure your system supports AVX open a terminal and run: `grep avx /proc/cpuinfo`. AVX should be listed under the flags for each CPU core. If nothing is returned it is most likely that your system does not support AVX. Technical users may be able to build an older version of TensorFlow \(1.13\) from source using the [instructions provided on their website](https://www.tensorflow.org/install/source). Alternatively you may use Mycroft with the PocketSphinx wake word engine; see (Switching Wake Word Listeners)[https://mycroft-ai.gitbook.io/docs/using-mycroft-ai/customizations/wake-word#switching-wake-word-listeners].
```

Comment:
added a ref at the end to the switching wake word listener page

NOTE: please fix inter-doc links (not sure if this is the correct way to link or not)

Reply:
This would totally work, but better to use:
`../customizations/wake-word#switching-wake-word-listeners`

I do want to switch this all over to a docs.mycroft.ai subdomain.

Reply:
ah yes whoops, relative paths ftw (fixed)

Reply:
fixed